react-native-libraries.json

```diff
@@ -13556,5 +13556,32 @@
     "android": true,
     "expoGo": false,
     "newArchitecture": true
-  }
+  },
+  {
+  "githubUrl": "https://github.com/benjamineruvieru/react-native-pdf-from-image",
+  "npmPkg": "react-native-pdf-from-image",
+  "examples": ["https://github.com/benjamineruvieru/react-native-pdf-from-image/tree/main/example"],
+  "images": ["https://www.benjamineruvieru.com/images/projects/react-native-pdf-from-image/logo.png"],
```

Comment:
```suggestion
```

We prefer not have a branding materials in the `images` array. Ideally those assets should show the package functionality:
* https://github.com/react-native-community/directory?tab=readme-ov-file#images